tests/models/xctests/converter/test_xcode_16_converter.py

```diff
@@ -0,0 +1,325 @@
+from datetime import datetime
+from datetime import timezone
+from typing import List
+from unittest import mock
+
+import pytest
+from codemagic.models.junit import Error
+from codemagic.models.junit import Property
+from codemagic.models.junit import Skipped
+from codemagic.models.junit import TestCase
+from codemagic.models.junit import TestSuites
+from codemagic.models.xctests import XcResultTool
+from codemagic.models.xctests.converter import Xcode16XcResultConverter
+
+
+def _get_test_report_tests(_):
+    return {
+        "devices": [
```

Comment:
consider charge those dicts from an external json file

Reply:
Changed in [8ced5b1](https://github.com/codemagic-ci-cd/cli-tools/pull/431/commits/8ced5b1e6ff6ed3cb6f438ad43c94e97d2cceec5).